R/utils-time.R

```diff
@@ -4,7 +4,7 @@
 #' 
 #' @description A convienence function that will attempt to parse a (day
 #'   point-type) date, UTC time, or offset time extremes JSON.
-#' @param x A character vector of the date/time.
+#' @param x A vector of character vectors of the dates/times.
```

Comment:
the prev documentation did make sense - "a vector of vectors" is a little more confusing. To be completely clear, I might say "A character vector of length one or greater of the dates/times."